src/internal/kopia/upload.go

```diff
@@ -1061,6 +1156,9 @@ const (
 	// statSkipMerge denotes the number of directories that weren't merged because
 	// they were marked either DoNotMerge or New.
 	statSkipMerge = "directories_skipped_merging"
+	// statPruned denotes the number of subtree roots that selective subtree
+	// pruning applied to.
+	statPruned = "subtrees_pruned"
```

Comment:
nit: Should we add a TODO here? I don't see `statPruned` being modified in this PR.

Reply:
oh whoops, good catch! I'll just add that in real quick. It'll just be another branch off one of the if-blocks